File: Level1/7.같은숫자는싫어/7.같은숫자는싫어.cpp
#include <vector>
#include <iostream>

using namespace std;

vector<int> solution(vector<int> arr) 
{
    vector<int> answer;
    int nowNumber = -1;
    for(int i = 0 ; i<arr.size() ; i++){
        if(arr[i]!=nowNumber){
            nowNumber = arr[i];
            answer.push_back(arr[i]);
        }
    }
    
    return answer;
}

File: Level1/7.같은숫자는싫어/7.같은숫자는싫어.java
import java.util.*;

public class Solution {
    public int[] solution(int []arr) {
        ArrayList<Integer> answerAL = new ArrayList<Integer>();
        int nowNumber = -1;
        for(int i = 0 ; i<arr.length ; i++){
            if(arr[i]!=nowNumber){
                nowNumber = arr[i];
                answerAL.add(arr[i]);
            }
        }
        int[] answer = new int[answerAL.size()];
        for(int i = 0 ; i<answerAL.size() ; i++){
            answer[i] = answerAL.get(i).intValue();
        }
        

        return answer;
    }
}